File: app/src/main/java/com/example/review/New/CountList.java
package com.example.review.New;

public class CountList {
    public int   corrCount;
    public int   errCount;
    public int   totalNum;
    public float corrRate;
    public int   needCorrNum;

    public CountList() {

    }

    public CountList(int corrCount, int errCount, int totalNum) {
        this.corrCount = corrCount;
        this.errCount = errCount;
        this.totalNum = totalNum;
    }

    public void copyOf(CountList countList) {
        this.corrCount = countList.corrCount;
        this.errCount = countList.errCount;
        this.totalNum = countList.totalNum;
    }
}

File: app/src/main/java/com/example/review/New/CountList.kt
package com.example.review.New

class CountList {
    var corrCount = 0
    var errCount = 0
    var totalNum = 0
    var corrRate = 0f
    var needCorrNum = 0

    constructor() {}
    constructor(corrCount: Int, errCount: Int, totalNum: Int) {
        this.corrCount = corrCount
        this.errCount = errCount
        this.totalNum = totalNum
    }

    fun copyOf(countList: CountList) {
        corrCount = countList.corrCount
        errCount = countList.errCount
        totalNum = countList.totalNum
    }
}